qa/suites/fs/workload/tasks/7-mds-use-global-snaprealm-seq-config

```diff
@@ -0,0 +1 @@
+.qa/cephfs/mds-use-global-snaprealm-seq-config/
```

Comment:
nit: just name the directory `snaprealm`?

Reply:
You mean `globalsnaprealm` ? because it displays in job description as `globalsnaprealm`?{yes}/{no}

Reply:
How about `use-snaprealm` directory name and then the yamls can be named global.yaml and subvol.yaml?

Reply:
:+1:

Reply:
Done

Reply:
This is getting closer to getting run through qa suite. We will have to ensure that the test coverage is good enough to be able to merge this. I will spend time tomorrow and do the needful.